packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -73,6 +73,8 @@ const router = mesh.addVirtualRouter('router', {
 ```
 
 The router can also be created using the constructor and passing in the mesh instead of calling the `addVirtualRouter()` method for the mesh.
+It is important to note when calling the `addVirtualRouter()` method for the mesh, router is created in the same stack where the mesh is called.
+On the other hands, when the router is created using the constructor, it is created in the current stack.
```

Comment:
Yeah, this is kind of self-evident, so I would drop this sentence.

Reply:
Well noted.